doctr/models/recognition/transformer/pytorch.py

```diff
@@ -51,7 +51,9 @@ def scaled_dot_product_attention(
 
     scores = torch.matmul(query, key.transpose(-2, -1)) / math.sqrt(query.size(-1))
     if mask is not None:
-        scores = scores.masked_fill(mask == 0, -1e9)
+        # NOTE: to ensure the ONNX compatibility, masked_fill works only with int equal condition
+        # amp fix: -6.55e4 instead of -1e9
+        scores = scores.masked_fill(mask == 0, -6.55e4)
```

Comment:
why this value?
Can't we do -5e4 instead ? If there is a specific reason, add a ref to the corresponding documentation :)

Reply:
changed to float('-inf') should be clear now without a comment :) -inf is the masked -> softmax = 0.0 and it is compatible with half

Reply:
-6.55e4 is the first value which results in 0 and can be casted internal to half